.gitlab-ci.yml

```diff
@@ -979,6 +978,19 @@ deploy_puppy:
     - export VERSION=$(inv version --url-safe)
     - aws s3 cp --region us-east-1 ./agent $S3_DSD6_URI/puppy/agent-$VERSION --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers full=id=3a6e02b08553fd157ae3fb918945dd1eaae5a1aa818940381ef07a430cf25732
 
+# invalidate cloudfront cache
+deploy_cloudfront_invalidate:
+  stage: deploy_invalidate
```

Comment:
since it's a different stage, can you put this under the last job of the `deploy` stage in the file? I think it's better to keep things "ordered"